views/home.erb

```diff
@@ -0,0 +1,19 @@
+<h1>Chitter Home.</h1>
+<ul>
+  <% @peeps.each do |peep| %>
+    <li><%= "#{peep.first_name} (#{peep.handle}): #{peep.peep}" %></li>
```

Comment:
It's a tiny thing, but I wonder if the naming convention could be a little clearer here? Maybe something like 'content' to avoid duplication in 'peep.peep', which is a little unclear at a glance